src/api/helpers/formatEntityData.js

```diff
@@ -0,0 +1,386 @@
+ * @function
+ * @param {object} author - an Author ORM entity
+ * @returns {object} an object containing the basic data of an Author entity.
+ * Each property defaults to null. If author is null or undefined, returns null.
+ *
+ * @example
+ * 		const author = await orm.func.entity.getEntity(orm, 'Author', bbid, relations);
+ *		getAuthorBasicInfo(author);
+ *		/* => {
+			"bbid": "86f86a31-7c51-49ed-af71-9523cca30265",
+			"beginArea": null,
+			"beginDate": "1904-03-23",
+			"defaultAlias": {
+				"aliasLanguage": "English",
+				"name": "H. Beam Piper",
+				"sortName": "Piper, H. Beam"
+			},
+			"disambiguation": null,
+			"endArea": null,
+			"endDate": "1964-11-06",
+			"ended": true,
+			"gender": "Male",
+			"type": "Person"
+		}
+ */
+
+export function getAuthorBasicInfo(author: object) {
+	return _.isNil(author) ? null :
+		{
+			bbid: _.get(author, 'bbid', null),
+			beginArea: _.get(author, 'beginArea.name', null),
+			beginDate: _.get(author, 'beginDate', null),
+			defaultAlias: getDefaultAlias(author),
+			disambiguation: _.get(author, 'disambiguation.comment', null),
+			endArea: _.get(author, 'endArea.name', null),
+			endDate: _.get(author, 'endDate', null),
+			ended: _.get(author, 'ended', null),
+			gender: _.get(author, 'gender.name', null),
+			type: _.get(author, 'authorType.label', null)
+		};
+}
+
+/**
+ * A function to extract the basic details of a Publisher ORM entity
+ * @function
+ * @param {object} publisher - a Publisher ORM entity
+ * @returns {object} an object containing the basic data of a publisher entity.
+ * Each property defaults to null. If publisher is null or undefined, returns null.
+ *
+ * @example
+ * 		const author = await orm.func.entity.getEntity(orm, 'Publisher', bbid, relations);
+ *		getPublisherBasicInfo(publisher);
+ *		/* => {
+			"area": "India",
+			"bbid": "e418874e-5684-4fe9-9d2d-1b7e5d43fd59",
+			"beginDate": "1943",
+			"defaultAlias": {
+				"aliasLanguage": "[Multiple languages]",
+				"name": "Bharati Bhawan",
+				"sortName": "Bhawan, Bharati"
+			},
+			"disambiguation": null,
+			"endDate": null,
+			"ended": false,
+			"type": "Publisher"
+		}
+ */
+
+export function getPublisherBasicInfo(publisher: object) {
+	return _.isNil(publisher) ? null :
+		{
+			area: _.get(publisher, 'area.name', null),
+			bbid: _.get(publisher, 'bbid', null),
+			beginDate: _.get(publisher, 'beginDate', null),
+			defaultAlias: getDefaultAlias(publisher),
+			disambiguation: _.get(publisher, 'disambiguation.comment', null),
+			endDate: _.get(publisher, 'endDate', null),
+			ended: _.get(publisher, 'ended', null),
+			type: _.get(publisher, 'publisherType.label', null)
+		};
+}
+
+/**
+ * getEntityAliases is a function to extract the list of aliases from ORM entity
+ *
+ * @param {object} entity - An ORM Entity
+ * @returns {object} an object conatining the bbid and list of aliases data of an ORM entity
+ * Each property of object defaults to null. If entity is null or undefined, returns null.
```

Comment:
'Each property of object defaults to null' should be replaced with "Returns an empty array if no aliases were found"